samples/AnomalyDetector/tspCodeModel.json

```diff
@@ -21,14 +20,14 @@
      "Value": "v1.1"
     }
    ],
+   "CrossLanguageDefinitionId": "AnomalyDetector.APIVersion",
```

Comment:
OK is this change from the emitter from microsoft/typespec side?
if so could you open a PR on that side to revert this? This is quite unnecessary and may spam other changes.
We should expect there was nothing changed here in this PR.